blockchain/vm/contracts.go

```diff
@@ -456,7 +453,7 @@ func (c *validateSender) Run(input []byte, contract *Contract, evm *EVM) ([]byte
 	return []byte{1}, nil
```

Comment:
Why doesn't this return the error? @yoomee1313

Reply:
@KimKyungup `vmLog` is designed to return 0 if sender validation is failed, instead returning error. It return 1 when validation is succeed.